pkgs/applications/window-managers/sway/swayest-workstyle.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "swayest-workstyle";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Lyr-7D1h";
+    repo = "swayest_workstyle";
+    rev = version;
+    sha256 = "sha256-LciTrYbmJV0y0H6QH88vTBXbDdDSx6FQtO4J/CFLjtY=";
+  };
+
+  cargoSha256 = "sha256-34Ij3Hd1JI6d1vhv1XomFc9SFoB/6pbS39upLk+NeQM=";
+
+  doCheck = false; # No tests
+
+  meta = with lib; {
```

Comment:
This seems to be specific to wayland/sway. Platform should be set to `linux` only.

Reply:
Done!

Reply:
Addressed by latest rev which has this:
https://github.com/NixOS/nixpkgs/compare/8eb5b59515647e40cbddea5e1f88070511cf36a3..c7314009b3c556ea132c3046930ac5b4ac523a5f
Let me know if you see anything else. Thanks!